src/plugins/foxPage/components/OtherOpportunities/OtherOpportunities.tsx

```diff
@@ -0,0 +1,58 @@
+import { Flex } from '@chakra-ui/layout'
+import { Accordion, Text as CText } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { Card } from 'components/Card/Card'
+import { Text } from 'components/Text/Text'
+
+import { FoxOpportunities } from '../../FoxCommon'
+import { FoxOtherOpportunityPanel } from './FoxOtherOpportunityPanel'
+
+type OtherOpportunitiesProps = {
+  description: string
+  opportunities: FoxOpportunities
+}
+
+export const OtherOpportunities: React.FC<OtherOpportunitiesProps> = ({
+  description,
+  opportunities,
+}) => {
+  const translate = useTranslate()
+  const hasLiquidityPools = opportunities.liquidityPools?.length
+  const hasFarming = opportunities.farming?.length
+  const hasBorrowingAndLending = opportunities.borrowingAndLending?.length
+
+  return (
+    <Card display='block' width='full' borderRadius={8}>
+      <Card.Header pb={0}>
+        <Flex flexDirection='row' alignItems='center' mb={2}>
+          <CText fontWeight='bold' color='inherit'>
+            {translate('plugins.foxPage.otherOpportunitiesTitle')}
+          </CText>
+        </Flex>
+        <Text translation={description} color='gray.500' />
+      </Card.Header>
+      <Card.Body pb={0}>
+        <Accordion mx={-6} defaultIndex={[0]} allowToggle allowMultiple>
+          {hasLiquidityPools && (
+            <FoxOtherOpportunityPanel
+              opportunities={opportunities.liquidityPools}
+              title='plugins.foxPage.liquidityPools'
+            />
+          )}
+          {hasFarming && (
+            <FoxOtherOpportunityPanel
+              opportunities={opportunities.farming}
+              title='plugins.foxPage.farming'
+            />
+          )}
+          {hasBorrowingAndLending && (
+            <FoxOtherOpportunityPanel
+              opportunities={opportunities.borrowingAndLending}
+              title='plugins.foxPage.borrowingAndLending'
+            />
+          )}
```

Comment:
@gomesalexandre do you have an idea about how we could avoid that duplication? It's mainly because of the title, but I didn't want to add this key into the mocked data

Reply:
Looks good to me for now, but in the future we could make chunks (in that case, effectively a tuple of the 3 opportunities types LP, farming, and borrowing/lending) and map over them, WDYT?

Reply:
containing the title? I would be ok, why not do it there directly?

Reply:
Not really, IMO title shouldn't belong there, more like having the opportunity type in `OtherOpportunity`. Then, if we have a mapping of opportunity type to full type info  name, base URL etc), it would be as simple as a lookup in that mapping

Reply:
Oh yeah alright, looks fine to me, I'll do this inside this PR, it will be quick and avoid pushing legacy things 👍